blockchain/test/physical_address_claim_spec.js

```diff
@@ -1,6 +1,9 @@
-const PhysicalAddressClaim = artifacts.require('PhysicalAddressClaim');
+const PhysicalAddressClaim = artifacts.require('mocks/PhysicalAddressClaimMock');
```

Comment:
This is a workaround a bug in `solidity-coverage` [described here](https://github.com/sc-forks/solidity-coverage/issues/234#issuecomment-390846584).

Reply:
... the changes made for the woraround are all located in one [commit](https://github.com/poanetwork/poa-popa/pull/209/commits/df5b11d82f4864a3d877a8b999ddefa083188241).